src/components/Card.js

```diff
@@ -0,0 +1,75 @@
+export default class Card {
+  //good to go 4.30, idk what the reviewer means about changing profile elements in here
+  constructor(
+    data,
+    cardSelector,
+    handleImageClick,
+    handleDeleteCard,
+    handleLike
+  ) {
+    this._data = data;
+    this._name = data.name;
+    this._link = data.link;
+    this._cardSelector = cardSelector;
+    this._handleImageClick = handleImageClick;
+    this._handleDeleteCard = handleDeleteCard;
+    this._handleLike = handleLike;
+    this.id = data._id;
+    this.isLiked = data.isLiked;
+  }
+
+  _setEventListeners() {
+    this._likeButton.addEventListener("click", () => {
+      this._handleLike(this);
+    });
+    this._deleteButton.addEventListener("click", () => {
+      this._handleDeleteCard(this);
+    });
+    this._cardElement
+      .querySelector(".card__image")
+      .addEventListener("click", () => {
+        this._handleImageClick({ name: this._name, link: this._link });
+      });
+  }
+
+  handleLikeIcon() {
```

Comment:
Needs correction: pleasde remove this method, you don't need it anymore, use `updateIsLiked` instead